### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by Theodoraldo**
1. Add layout to handle pages.
2. Add navigation
3. Add sidebar navigation

### CODE REVIEW

File: src/components/layout/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                  BookShelf Library
+                </span>
+              </div>
+            </div>
+            <img src={user} alt="user logo" width={40} />
+          </div>
+        </div>
+      </nav>
+    </React.Fragment>
+  );
```
Comment (AnsarIbrahim) [inline src/components/layout/Navbar.js]: Change all class To className

**Review by AnsarIbrahim — COMMENTED** [review]
@Theodoraldo 

This is really nice Layout I like it 🔥🔥🔥

And Change all syntax to es6 👍

File: src/components/layout/Navbar.js (hunk 1 above)
Comment (Theodoraldo) [reply src/components/layout/Navbar.js]: Sure

**Review by AnsarIbrahim — APPROVED** [review]
@Theodoraldo 

Approved Great 👍

**Pull request merged**